nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/pipe/AbstractListenerPipeConsumer.java

```diff
@@ -55,4 +55,8 @@ protected void processEventBundles(List<EventBundle> bundles) {
         }
     }
 
+    public boolean waitForCompletion(long timeoutMillis) throws InterruptedException {
```

Comment:
Why the waitForCompletion is not part of the PipeConsumer interface like for EBPipe or EBDispatcher ?